docs/DesignDocs/InitTimeConfigurationInterface.md

```diff
@@ -0,0 +1,722 @@
+`oe_result_t oe_get_evidence(...)` function sets an integrity algorithm ID not
+supported by the Verifier Plugin, the Verifier Plugin should output the
+`inittime_custom_claims_buffer` as it is, as an unverified init-time claim. In
+that case, the consumer of the outputted claims is responsible to verify the
+integrity of the `inittime_custom_claims_buffer` claim using the base claim of
+config_id/config_svn.
+
+```C
+/**
+ * oe_get_evidence
+ *
+ * Generates the evidence for the given format id.
+ * This function is only available in the enclave.
+ *
+ * @param[in] format_id The format ID of the evidence to be generated.
+ * @param[in] flags A bit-wise parameter. Currently there is one bit
+ * defined: OE_EVIDENCE_FLAGS_EMBED_FORMAT_ID. If this bit is set,
+ * the evidence and endorsements will be wrapped with a header containing
+ * the format ID.
+ * @param[in] runtime_custom_claims_buffer The optional runtime custom claims
+ * buffer. When provided, the content of the buffer is included in the
+ * evidence_buffer, with integrity protection. Depending on the underlining TEE
+ * and the plugin implementation, the content might or might not be further
+ * encrypted.
+ * @param[in] runtime_custom_claims_buffer_size The number of bytes in the
+ * runtime custom claims buffer.
+ * @param[in] inittime_custom_claims_buffer The optional inittime custom claims
+ * buffer. When provided, the content of the buffer is included in the
+ * evidence_buffer as plaintext. The integrity protection of the content is
+ * provided by the underlying TEE and the enclave SW outside the plugin.
+ * @param[in] inittime_custom_claims_buffer_size The number of bytes in the
+ * inittime custom claims buffer.
+ * @param[in] optional_parameters The optional format-specific input parameters.
+ * @param[in] optional_parameters_size The size of optional_parameters in bytes.
+ * @param[out] evidence_buffer An output pointer that will be assigned the
+ * address of the dynamically allocated evidence buffer.
+ * @param[out] evidence_buffer_size A pointer that points to the size of the
+ * evidence buffer in bytes.
+ * @param[out] endorsements_buffer If not NULL, an output pointer that will be
+ * assigned the address of the dynamically allocated endorsements buffer.
+ * @param[out] endorsements_buffer_size A pointer that points to the size of the
+ * endorsements buffer in bytes.
+ * @retval OE_OK The function succeeded.
+ * @retval OE_INVALID_PARAMETER At least one of the parameters is invalid.
+ * @retval OE_NOT_FOUND The input evidence format id is not supported.
+ * @retval other appropriate error code.
+ */
+oe_result_t oe_get_evidence(
+    const oe_uuid_t* format_id,
+    uint32_t flags,
+    const void* runtime_custom_claims_buffer,
+    size_t runtime_custom_claims_buffer_size,
+    const oe_inittime_custom_claim_buffer_t* inittime_custom_claims_buffer,
+    size_t inittime_custom_claims_buffer_size,
```

Comment:
Avoid abbreviations and put underscore between words.
"inittime" does not follow OE style guidelines.
should be `initialization_time` unless you use a shorter phrase or acronym